examples/normalize_nodes/messages/01_premerged_message.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
Should this file be names something like `01_prenormalized_message.json` instead of `premerged`?